sources/Cargo.lock

```diff
@@ -94,9 +94,9 @@ dependencies = [
 
 [[package]]
 name = "actix-http"
-version = "3.8.0"
+version = "3.9.0"
```

Comment:
Can you do a more targeted update for just `tough`? I prefer the bulk crate updates to come in a separate PR so it's easier to revert if it causes trouble.

Reply:
Sure. Created a new PR: https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/136